stepactions/gather-cluster-resources/0.1/gather-cluster-resources.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: tekton.dev/v1alpha1
+kind: StepAction
+metadata:
+  name: gather-cluster-resources
+spec:
+  description: >-
+    This StepAction executes the gather extra script that gathers konflux related resources from the cluster.
+    It also allows for a URL with another gather job to be passed to gather custom resources.
+  image: quay.io/konflux-qe-incubator/konflux-qe-tools:latest
+  params:
+    - name: credentials
+      type: string
+      description: A volume containing credentials to the remote cluster
+      default: ""
```

Comment:
I think this should be actually a required value because it's used in `volumeMounts`, which requires a valid volume name.

We should document that if users want to use `oc-login-command`, they could just pass the valid name of the volume that is a type of emptyDir.

Or if it sounds too complicated, other option is to have 2 StepActions:
1. For using oc-login-command
2. For using KUBECONFIG from secret

Reply:
tbh I feel it's fine to just have the users pass an emptyDir since it won't really be used. Just an empty mount. 
Having two steps is fine but a lot of repetition IMO. Up to you, I can separate it too.